### PR TITLE
[#IC-182] Remove errorMessage from CIE redirect error url params

### PR DESCRIPTION
**Pull request by BurnedMarshal**
#### List of Changes

The `ClientErrorRedirectionUrl` for minors of 14 yo returns only the query param `errorCode`. `errorMessage` was removed. The newer app versions will handle all the query params sent with the redirect url pagopa/io-app#3642

#### Motivation and Context

The actual version of the IO App cannot handle properly the error when `errorMessage` prop is defined with `errorCode`.

#### How Has This Been Tested?

Unit test

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/controllers/authenticationController.ts [hunk 1]
```diff
@@ -123,10 +125,11 @@ export default class AuthenticationController {
       this.hasUserAgeLimitEnabled &&
       !isOlderThan(AGE_LIMIT)(parse(spidUser.dateOfBirth), new Date())
     ) {
-      const redirectionUrl = this.getClientErrorRedirectionUrl(
-        AGE_LIMIT_ERROR_MESSAGE,
-        AGE_LIMIT_ERROR_CODE
-      );
+      // The IO App show the proper error screen if only the `errorCode`
+      // query param is provided and `errorMessage` is missing.
```
Comment (Undermaken) [inline src/controllers/authenticationController.ts:129]:
```suggestion
      // The IO App show the proper error screen if only the `errorCode`
      // query param is provided and `errorMessage` is missing.
      // this constrain could be ignored when this PR https://github.com/pagopa/io-app/pull/3642 is merged and 
      // released in a certain app version
```
I don't know which is your approach to keep track of future change. Mine is just a proposal

**Review by Undermaken — APPROVED** [review]

File: src/controllers/authenticationController.ts (hunk 1 above)
Comment (BurnedMarshal) [reply src/controllers/authenticationController.ts:129]: Ok, i just extend that comment with `and that version become the minimum version supported.` We can even refers this PR with the IO App PR into the description.

Comment (BurnedMarshal) [reply src/controllers/authenticationController.ts:129]: Fixed in a63459f

File: src/config.ts [hunk 2]
```diff
@@ -447,17 +448,40 @@ export const getClientProfileRedirectionUrl = (
...
+    })
+  ]),
+  t.intersection([
+    t.partial({
+      errorMessage: NonEmptyString
+    }),
+    t.interface({
+      errorCode: t.number
+    })
+  ]),
```
Comment (fabriziopapi) [inline src/config.ts:468]:
```suggestion

```
I think we could remove this block.

Comment (BurnedMarshal) [reply src/config.ts:468]: Why? If we remove this block we cannot call this
```ts
this.getClientErrorRedirectionUrl({
  errorCode: AGE_LIMIT_ERROR_CODE
});
```

Comment (fabriziopapi) [reply src/config.ts:468]: you are right, I missed it.

**Review by fabriziopapi — APPROVED** [review]

**Pull request merged**